packages/terra-consumer-nav/src/components/modal/Modal.jsx

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I think this is the master plan in the end (same with card and popup). This is here for now for speed